homeassistant/components/conversation/__init__.py

```diff
@@ -264,6 +280,121 @@ async def async_handle_intents(
     return await agent.async_handle_intents(user_input, intent_filter=intent_filter)
 
 
+async def async_get_intents(
+    hass: HomeAssistant, language: str, source: IntentSource = IntentSource.ALL
+) -> Intents:
+    """Load intents for a language."""
```

Comment:
```suggestion
    """Collect available intents for a given language.

    Returns a collection of intents.
    """
```
Specify return param in comment.